js/src/redux/middleware.js

```diff
@@ -30,7 +31,8 @@ export default function (api) {
   const middleware = [
     settings.toMiddleware(),
     signer.toMiddleware(),
-    errors.toMiddleware()
+    errors.toMiddleware(),
+    certificationsMiddleware(api)
```

Comment:
Any reason for not following the conventions, i.e. `toMiddleware()` as per the rest? (Took a long time to get all the middleware grouped an following a pattern.)